src/pages/registration/ContributorsPanel.tsx

```diff
@@ -1,10 +1,11 @@
+import { ErrorMessage, FieldArray, FieldArrayRenderProps, useFormikContext } from 'formik';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { FormHelperText, Typography } from '@material-ui/core';
-import { useFormikContext, FieldArray, ErrorMessage, FieldArrayRenderProps } from 'formik';
-import Card from '../../components/Card';
-import { Registration } from '../../types/registration.types';
+import BackgroundDiv from '../../components/BackgroundDiv';
+import theme from '../../themes/mainTheme';
```

Comment:
du nevnte `useTheme` tidligere, så nå begynner jeg å lure...
Feks bruker vi alltid `useTranslation()`, men det funker jo like godt å importere vår `i18n` på samme måte som her også?

Må nok bruke hooken for at komponenten skal rendres på nytt pga endring i theme/translation, men det vil jo sjelden/aldri skje i di tilfellene, så hva er vitsen? eller fører disse importene til at det opprettes et nytt theme-objekt hver gang, i stedet for å bruke det objektet som allerede er der? 😕

Reply:
godt spørsmål 😬  det samme skjer jo sikkert her. Vet ikke hvor mye tid vi mister på å gjøre det sånn da men... Er ganske sikker på at imports ikke fører til nytt theme-objekt ja. Det er jo en import av en eksisterende ting, ikke en opprettelse.

Reply:
det er kanskje bare om man trenger rerending man trenger å bruke hooken da 🤷‍♂️